libcodechecker/analyze/analyzers/analyzer_clang_tidy.py

```diff
@@ -136,6 +136,10 @@ def construct_analyzer_cmd(self, result_handler):
 
             analyzer_cmd.extend(self.buildaction.compiler_includes)
 
+            if not next((x for x in analyzer_cmd if x.startswith('-std=')),
```

Comment:
gcc accepts -std=c++11 in the following forms
-std=c++11
--std=c++11
--std c++11

Only the first form is handled.
Please handle the other variations too.